src/assets/scss/base/_settings.scss

```diff
@@ -0,0 +1,10 @@
+$grey: #232323;
```

Comment:
Lets use variables like $primary, $secondary, $accent. So if you want to change the colours at any point, their not named the exact colour.  $white is fine though.